code/modules/mob/mob.dm

```diff
@@ -25,6 +25,7 @@ var/next_mob_id = 0
 	..()
 
 /atom/proc/prepare_huds()
+	hud_list = list()
```

Comment:
I'll look over these tomorrow, i only did quick "can i see huds at all" test of this so far.